Kronos-Shifts-Connector/Microsoft.Teams.Shifts.Integration/Microsoft.Teams.Shifts.Integration.API/Controllers/ShiftController.cs

```diff
@@ -706,39 +706,33 @@ private Shift GenerateTeamsShiftObject(UserDetailsModel user, UpcomingShiftsResp
             // create entries from not found list
             for (int i = 0; i < notFoundShifts.Count; i++)
             {
-                var requestString = JsonConvert.SerializeObject(notFoundShifts[i]);
-
                 var httpClient = this.httpClientFactory.CreateClient("ShiftsAPI");
-                httpClient.DefaultRequestHeaders.Authorization = new AuthenticationHeaderValue("Bearer", configurationDetails.ShiftsAccessToken);
-                httpClient.DefaultRequestHeaders.Accept.Add(new MediaTypeWithQualityHeaderValue("application/json"));
+                httpClient.DefaultRequestHeaders.Authorization = new AuthenticationHeaderValue("Bearer", configurationDetails.GraphConfigurationDetails.ShiftsAccessToken);

```

Comment:
Could this be moved to the utility method seeing as you are passing what is needed to do so?